aliasing/api/functions.py

```diff
@@ -247,3 +248,17 @@ def verify_signature(ctx: disnake.ext.commands.Context, data: str):
         "channel": channel and AliasChannel(channel),  # may be None
         "author": author and AliasAuthor(author),  # may be None
     }
+
+
+def parse_coins(args: str) -> dict:
```

Comment:
@Croebh I'm not sure if autodoc will catch this - can you check that it made it in?